app/views/vendors/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1>Add a Vendor</h1>
+
+<section>
+  <%= form_for @vendor do |f| %>
```

Comment:
Again, I think if we did `form_for [@market, @vendor] do |f|` this would still work, and we wouldn't need to use the hidden field below.